community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -0,0 +1,4106 @@
+{
```

Comment:
<p>Do we need this for this particular sample? I think we should simplify the notebook as much as possible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='126'/>

Reply:
Removed the training parts from this notebook.